Neos.Flow/Classes/ObjectManagement/Proxy/ProxyMethodGenerator.php

```diff
@@ -174,4 +242,78 @@ protected function buildCallParentMethodCode(string $fullClassName, string $meth
         }
         return 'parent::' . $methodName . '(' . $this->buildMethodParametersCode($fullClassName, $methodName, false) . ");\n";
     }
+
+    /**
+     * Build the code for the attributes of a given \ReflectionMethod object.
+     *
+     * Note: This is just a preliminary solution until https://github.com/laminas/laminas-code/pull/145
+     *       is implemented and released.
+     *
+     * @param \ReflectionMethod $reflectionMethod The \ReflectionMethod object to retrieve attributes from.
+     * @return string The code for the attributes of the given \ReflectionMethod object.
+     */
+    protected function buildAttributesCode(\ReflectionMethod $reflectionMethod): string
+    {
+        $indent = $this->getIndentation();
+        $attributesCode = "";
+
+        foreach ($reflectionMethod->getAttributes() as $attribute) {
+            $attributeName = "\\" . ltrim($attribute->getName(), '\\');
+            $argumentsString = $this->formatAttributesArguments($attribute->getArguments());
+            $attributesCode .= "{$indent}#[{$attributeName}({$argumentsString})]" . self::LINE_FEED;
+        }
+
+        return $attributesCode;
+    }
+
+    /**
+     * Formats the arguments of attributes into a string.
+     *
+     * @param array $arguments An array of arguments for attributes.
+     *
+     * @return string The formatted arguments as a string.
+     */
+    private function formatAttributesArguments(array $arguments): string
+    {
+        $formattedArguments = [];
+
+        foreach ($arguments as $key => $value) {
+            $formattedArguments[] = "{$key}: " . $this->formatAttributeValue($value);
```

Comment:
According to the docs something like this here is valid:

`#[JsonSerialize('foobar')]`

so we probably need to have a test for this kind of value without key and properly handle it here.

Reply:
Thanks! I added support for that, too.